Solutions/Marain.TenantManagement.Cli/Marain/TenantManagement/Cli/Commands/CreateClientTenantCommand.cs

```diff
@@ -27,16 +27,21 @@ public CreateClientTenantCommand(ITenantManagementService tenantManagementServic
                 Description = "The name of the new client.",
                 Arity = ArgumentArity.ExactlyOne,
             };
+            var parentId = new Option<string>("--parentId")
+            {
+                Description = "[Optional] The ID of the parent Client Tenant.",
```

Comment:
You probably don't need `[Optional]` in here - my understanding is that options are all optional - when you get the help, they end up looking like this:

```
Usage:
  marain enroll [options] <clientTenantId> <serviceTenantId>

Arguments:
  <clientTenantId>     The Id of the client tenant.
  <serviceTenantId>    The Id of the service tenant.

Options:
  --config <config>    JSON configuration file to use when enrolling.
  -?, -h, --help       Show help and usage information
```

Reply:
Removed